CNKI.js

```diff
@@ -262,25 +258,10 @@ function scrape(ids, doc, url, itemInfo) {
 			//	newItem.extra = 'CN ' + newItem.callNumber;
 				newItem.callNumber = "";
 			}
-			// 018<lyb018@gmail.com>: 20210108
-			var num = doc.querySelector('#func3 .num');
-			if (!num) {
-				var e = doc.querySelector('[value="' + dbname + '!' + filename + '!' + curRec + '!' + queryID + '"]');
-				if (e) {
-					e = e.parentElement;
-				}
-				if (e) {
-					e = e.parentElement;
-				}
-				if (e) {
-					e = e.querySelector('.quote a');
-				}
-				if (e) {
-					num  = e
-				}
-			}
-			if (num) {
-				newItem.extra = num.textContent.replace(/[^\d*]/g, '');
+			var numEle = doc.querySelector('#func3 .num');
+			if (numEle) {
+				Z.debug(numEle.textContent.replace(/[^\d*]/g, ''))
```

Comment:
I don't think the `Z.debug` line needs to be here, but otherwise this looks much better, thanks!